ros_network_viz/ros_network_viz.py

```diff
@@ -111,6 +124,21 @@ def update_path(self, source_point, target_point):
 
         self.setPath(path)
 
+    # PyQt method override
```

Comment:
One more minor nit: I tend to keep all of the overridden PyQt methods grouped together near the beginning of the class definition.  I just find it easier to read and find the Qt stuff (vs application-specific methods).  So I'll suggest moving this up above `update_path`.

Reply:
Sure; moved as suggested.